github_connector_odoo/models/odoo_module_version.py

```diff
@@ -439,6 +441,9 @@ def manifest_2_odoo(self, info, repository_branch, module):
             "description_rst": info["description"],
             "external_dependencies": info.get("external_dependencies", {}),
             "author": ",".join([x.strip() for x in sorted(author_list) if x.strip()]),
+            "maintainers": ",".join(
+                [x for x in sorted(info.get("maintainers", [])) if x]
+            ),
```

Comment:
Why is the list comprehension needed? Wouldnt be this the same?:
```suggestion
            "maintainers": ",".join(info.get("maintainers", []))),
```